csrc/scheduler/matmul.cpp

```diff
@@ -711,6 +711,17 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
       mma_ops.size() == 1,
       "scheduleMatmul supports fusion with single mma op in definition, got ",
       mma_ops.size());
+
+  // Skip scheduling if Matmul will be expression evaluated.
+  if (isOptionEnabled(EnableOption::MatmulExprEval)) {
+    NVF_CHECK(fusion->outputs().size() == 1)
+    fusion->markOutputForEvaluation(fusion->outputs()[0]);
+    scheduler_debug_utils::log(
+        __FUNCTION__,
```

Comment:
Nice! This is an easy way to add a prefix to distinguish debug messages from different schedulers. 

Nit: Would `__FILE__` be better? We might want to move this call around as part of a refactoring for example. If it's moved into a function whose name doesn't contain matmul and/or whose name is not unique across the nvFuser code base, it's going to be hard to tell which file this message comes from. To that extent, I think `__FILE__` is slightly better as a way to locate the source of a message.

Reply:
Good point: Will change it to `__FILE__ : __LINE__`.

Reply:
That's even better. FWIW, for the future, we could create a macro around `scheduler_debug_utils::log` so we won't have to type `__FILE__` and `__LINE__` for convenience.